fastestimator/network.py

```diff
@@ -768,8 +772,16 @@ def _fe_compile(model: Model,
         model.current_optimizer = optimizer_fn
     model.optimizer = optimizer_fn
     model.fe_compiled = True
+
     if weight:
+        if not os.path.isfile(weight):
+            tmp_dir = tempfile.mkdtemp()
```

Comment:
weren't we going to combine this with the ~/.fastestimator_data directory to cache the downloads?